### PR TITLE
Easier ant prevention

### PR DESCRIPTION
**Pull request by nova2053**
## Labels
[CATERING] [QoL]

## About the PR 
Food items now have a one minute immunity to ants after having their loc changed.

## Why's this needed?
It's annoying to open a crate or fridge full of food for just a couple seconds and have a whole bunch of ants spawn.
 

## Changelog 

```changelog
(u)nova2053
(+)Food items now have a one minute immunity to ants after being moved.
```

### CODE REVIEW

**Review by ZeWaka — COMMENTED** [review]
Seems a bit sus, I'd rather have some sort of delay before ants spawn when loc is changed

**Comment by nova2053** [comment]
> Seems a bit sus, I'd rather have some sort of delay before ants spawn when loc is changed

that could work, though i would still like to keep the functionality of racks preventing ants, because more food storage options are nice and it could be useful for stuff like walk-in freezers in the future

**Comment by ZeWaka** [comment]
> > Seems a bit sus, I'd rather have some sort of delay before ants spawn when loc is changed
> 
> that could work, though i would still like to keep the functionality of racks preventing ants, because more food storage options are nice and it could be useful for stuff like walk-in freezers in the future

ants should just not spawn in cold temps

**Comment by nova2053** [comment]
> > > Seems a bit sus, I'd rather have some sort of delay before ants spawn when loc is changed
> > 
> > 
> > that could work, though i would still like to keep the functionality of racks preventing ants, because more food storage options are nice and it could be useful for stuff like walk-in freezers in the future
> 
> ants should just not spawn in cold temps

Makes sense, ill just remove the rack invulnerability to ants thing. also added the ant spawn delay like you recommended

**Review by ZeWaka — COMMENTED** [review]
Use `TIME`

**Comment by nova2053** [comment]
> Use `TIME`

done, changed all instances of `world.time` in that code to `TIME`

**Review by Azrun — APPROVED** [review]

File: code/modules/chemistry/tools/food_and_drink.dm [hunk 1]
```diff
@@ -184,8 +189,8 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks)
 		..()
 
 	process()
-		if (world.time - create_time >= 3 MINUTES)
-			create_time = world.time
+		if ((TIME - create_time >= 3 MINUTES) && (TIME - time_since_moved >= 1 MINUTES))
```
Comment (ZeWaka) [inline code/modules/chemistry/tools/food_and_drink.dm:192]:
```suggestion
		if ((TIME - create_time >= 3 MINUTES) && (TIME - time_since_moved >= 1 MINUTE))
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**